plutarch-docs/Concepts/GenericProgramming.md

```diff
@@ -22,6 +38,9 @@ Usually depends that given datatype derives `Generic` typeclass also or some oth
 
 ### generics-sop
 
+A really good introduction to `generics-sop` by the maker of the library, Andres Löh, can be found 
+[in this YouTube video recorded at the 2020 ZuriHac](https://www.youtube.com/watch?v=pwnrfREbhWY)
+
 Overall image of [generics-sop](https://github.com/well-typed/generics-sop) package.
 
 [Introduction by srid](https://github.com/srid/generics-sop-examples/blob/master/doc/draft.md).
```

Comment:
We should update this to: https://srid.ca/generics-sop-intro

Reply:
I don't think referring people to his personal website where he hosts his political opinions is a wise decision because I would prefer to keep that separated but if you think it is, I might perhaps update it.

Reply:
But the link is outdated (the file is no longer there). I didn't really know nor care about the existence of political opinions in the website; I don't know if other readers would/should care either.

Reply:
If it doesn't exists I'll link the other one.

Reply:
done